### PR TITLE
⚡️🧠 (Add outbrain onward component)

### PR DESCRIPTION
**Pull request by nicl**
Note, this will probably need some changes to accommodate other
onward work which @AWare has already done, so I'll handle the conflict when that gets merged.

There is at least one question here around compliance:

https://github.com/guardian/frontend/blob/master/common/app/views/fragments/amp/outbrain.scala.html#L11

I'm not sure what the AMP_1 and AMP_2 behaviour is about so will check with @NataliaLKB who implemented this.

screenshot 2018-12-19 at 17 58 53

TODOs:

* check on AMP_1 vs AMP_2 behaviour
* understand why overflow attribute doesn't get rendered at all and causes console error

### CODE REVIEW

**Comment by NataliaLKB** [comment]
Hmm. I don't remember who implemented it @nicl. Someone on commercial dev. Might be worth talking to @jeteve, @katebee, or @kelvin-chappell ?

**Comment by kelvin-chappell** [comment]
@NataliaLKB I think you're implicated in it: https://github.com/guardian/frontend/pull/16239

**Comment by nicl** [comment]
@kelvin-chappell @NataliaLKB yeah I had also checked the commit history which is why I flagged Natalia originally 😬

**Comment by NataliaLKB** [comment]
Haha! Hilarious. Thanks @kelvin-chappell! I (Very clearly)definitely don't remember the difference between AMP_1 or AMP_2. Sorry I can't be more helpful!

File: packages/frontend/amp/components/Onward.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <amp-iframe
+            key={outbrainURL}
+            height="480"
+            sandbox="allow-scripts allow-same-origin allow-popups"
+            layout="fixed-height"
+            frameborder="0"
+            src={outbrainURL}
+            class={outbrain}
+        >
+            <div overflow={true}>
```
Comment (nicl) [inline packages/frontend/amp/components/Onward.tsx]: The overflow attribute here causes an error currently and doesn't get rendered.

Comment (nicl) [reply packages/frontend/amp/components/Onward.tsx]: Any thoughts on why this happens appreciated!

Comment (SiAdcock) [reply packages/frontend/amp/components/Onward.tsx]: As discussed, this is a non-standard attribute (as far as React is concerned) so it's probably a bit confused. Perhaps try passing it as a string?

File: packages/frontend/amp/components/Onward.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { css } from 'react-emotion';
+import Plus from '@guardian/pasteup/icons/plus.svg';
+import InnerContainer from '@frontend/amp/components/InnerContainer';
+import { OnwardContainer } from '@frontend/amp/components/OnwardContainer';
```
Comment (SiAdcock) [inline packages/frontend/amp/components/Onward.tsx]: Some modules have default exports and some have named exports. I think we agreed to standardise on [named exports](https://github.com/guardian/dotcom-rendering/blob/master/docs/contributing/code-style.md#always-used-named-exports), does that work for you?

**Comment by nicl** [comment]
@SiAdcock thanks for the comments. I've fixed the overflow issue, although I will want to test amp validity at a later date. But otherwise, are you happy for me to merge?

**Comment by nicl** [comment]
I've merged this but actually there is more work to do around whether AMP is compliant. But it requires a bit of a think so though I'd merge this first so as to avoid conflicts/blocking other things progressing.

**Pull request merged**